images/base/scripts/common/setup.sh

```diff
@@ -83,7 +83,12 @@ fi
 
 # First install protoc
 cd /tmp
-git clone https://github.com/google/protobuf.git
+if [ x$MACHINE = xs390x ]
+then
+    git clone https://github.com/gongsu832/protobuf.git
```

Comment:
I'm not sure I follow you. The patch would be generated by git diff using my repo to start with. To look at the patch, you either download it or clone the repo and do a git diff.